armbian/base/build/customize-armbian-rockpro64.sh

```diff
@@ -369,15 +367,35 @@ EOF
 
 
 # LIGHTNING --------------------------------------------------------------------
+BIN_DEPS_TAG="v0.0.1-alpha"
 LIGHTNING_VERSION="0.7.0"
 
-cd /usr/local/src/
-git clone https://github.com/ElementsProject/lightning.git || true
-cd lightning
-git checkout v${LIGHTNING_VERSION}
-./configure
-make
-make install
+## either compile c-lightning from source (default), or use prebuilt binary
+if [ "${BASE_BUILD_LIGHTNINGD}" == "true" ]; then
+  apt install -y  autoconf automake build-essential git libtool libgmp-dev \
+                  libsqlite3-dev python python3 net-tools zlib1g-dev libsodium-dev
+
+  cd /usr/local/src/
+  git clone https://github.com/ElementsProject/lightning.git || true
+  cd lightning
+  git checkout v${LIGHTNING_VERSION}
+  ./configure
+  make -j 4
+  make install
+
+else
+  apt install -y libsodium-dev
```

Comment:
(Optional) Since we install `libsodium-dev` in both branches of this `if` statement, it's not really conditional, so we could also install it along with other dependencies higher up in the script and remove it from both branches.

Reply:
I'd like to keep it in the c-lightning section, but can move it in front of the if-clause.

Reply:
SGTM.